### PR TITLE
add c2 wofs styles and terria config

### PR DESCRIPTION
**Pull request by tom-butler**
:warning: Don't merge until the data has been indexed and ranges have been generated :warning:

### CODE REVIEW

**Comment by tom-butler** [comment]
Only includes statistics and wet count

File: services/ows.py [hunk 1]
```diff
@@ -1844,6 +1844,326 @@
...
+                #  not required by the standard.)
+                "default_style": "WOfS_frequency",
+            },
+            {
+                # Included as a keyword  for the layer
+                "label": "WOfS C2 Statistics",
+                # Included as a keyword  for the layer
+                "type": "Wet Count",
+                # Included as a keyword  for the layer
+                "variant": "25m",
```
Comment (Kirill888) [inline services/ows.py]: 30 not 25

File: services/ows.py [hunk 2]
```diff
@@ -1844,6 +1844,326 @@
...
+                # The WMS name for the layer
+                "name": "wofs_2_annual_summary_statistics",
+                # The Datacube name for the associated data product
+                "product_name": "ga_ls8c_wofs_2_annual_summary",
+                "abstract": """Water Observations from Space (WOfS) Statistics is a set of statistical summaries of 
+                the WOfS product which combines WOfS observations into summary products that help the understanding 
+                of surface water across Africa. WOfS Statistics is calculated from the full depth time series (1984 ? 
+                2018). The water detected for each location is summed through time and then compared to the number of 
+                clear observations of that location. The result is a percentage value of the number of times water 
+                was observed at the location. The layers available are: the count of dry observations; the count of
```
Comment (Kirill888) [inline services/ows.py]: there are no dry observations, there is a count of clear observation (no cloud/shadow)

File: services/ows.py [hunk 3]
```diff
@@ -1844,6 +1844,326 @@
...
+                "type": "Water Summary",
+                # Included as a keyword  for the layer
+                "variant": "30m",
+                # The WMS name for the layer
+                "name": "wofs_2_annual_summary_statistics",
+                # The Datacube name for the associated data product
+                "product_name": "ga_ls8c_wofs_2_annual_summary",
+                "abstract": """Water Observations from Space (WOfS) Statistics is a set of statistical summaries of 
+                the WOfS product which combines WOfS observations into summary products that help the understanding 
+                of surface water across Africa. WOfS Statistics is calculated from the full depth time series (1984 ?
```
Comment (Kirill888) [inline services/ows.py]: this product is Annual, not full history

**Pull request merged**